src/components/ChangePlan/ChangePlan.js

```diff
@@ -267,6 +267,8 @@ const ChangePlan = ({ location, dependencies: { planService, appSessionRef } })
       </Helmet>
       {state.loading ? (
         <Loading page />
+      ) : state.currentPlan?.type === 'exclusive' ? (
```

Comment:
Maybe something like this will be better, It's more simple: 
![image](https://user-images.githubusercontent.com/84402180/134030493-59f18f11-1f5c-4e08-974f-e36fbf829c2d.png)